rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -137,11 +139,12 @@ void voteFeePerKbChange_nullFeeThrows() {
         BridgeStorageProvider provider = mock(BridgeStorageProvider.class);
         Transaction tx = mock(Transaction.class);
         BridgeConstants constants = mock(BridgeConstants.class);
+        FeePerKbConstants feePerKbMainNetConstants = mock(FeePerKbMainNetConstants.class);
```

Comment:
it doesn't seem consistent to mock `BridgeConstants` but specific `FeePerKbMainnetConstants`.
Lets choose one